cmd/publish/main_test.go

```diff
@@ -12,7 +13,7 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/mock"
 )
-
+const FIXTURES_PATH string = "../../fixtures/policyAsCode"
```

Comment:
Hi @bhavyastar! I believe you can run `gofmt -s -w <path_to_file>.go` to format this file.

Reply:
Hello @noaabarki, I did the same thing, ma'am. It is still showing this. But, yes I can try again by running this command.